tests/src/test/scala/common/WhiskProperties.java

```diff
@@ -157,6 +172,10 @@ public static int numberOfInvokers() {
         return getInvokerHosts().length;
     }
 
+    public static boolean isSSLCheckRelaxed() {
+        return Boolean.valueOf(getPropFromSystemOrEnv("whisk.ssl.relax"));
```

Comment:
this is handy - could then use it here directly?
https://github.com/apache/incubator-openwhisk/blob/50de78a9d5f723485b0f568901189e2447580a47/tests/src/test/scala/system/rest/RestUtil.scala#L34